docs/data/date-pickers/accessibility/accessibility.md

```diff
@@ -0,0 +1,96 @@
+---
+productId: x-date-pickers
+title: Date and Time Pickers - Date format and localization
+githubLabel: 'component: pickers'
+packageName: '@mui/x-date-pickers'
+---
+
+# Date and Time Pickers - Accessibility
+
+<p class="description">The Date and Time Pickers have complete accessibility support, including built-in keyboard navigation that follows international standards.</p>
+
+## Guidelines
+
+The most commonly encountered conformance guidelines for accessibility are:
+
+- Globally accepted standard: [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
+- US:
+  - [ADA](https://www.ada.gov/) - US Department of Justice
+  - [Section 508](https://www.section508.gov/) - US federal agencies
+- Europe: [EAA](https://ec.europa.eu/social/main.jsp?catId=1202) (European Accessibility Act)
+
+WCAG 2.1 has three levels of conformance: A, AA, and AAA.
+Level AA meets the most commonly encountered conformance guidelines.
+This is the most common target for organizations so what we aims to support very well.
+
+<!-- Those examples do not cover the clock component -->
+
+The WAI-ARIA Authoring Practices includes examples on [Date Picker Dialog](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/datepicker-dialog/) and [Date Picker Spin Button](https://www.w3.org/WAI/ARIA/apg/patterns/spinbutton/examples/datepicker-spinbuttons/) that provide valuable information on how to optimize the accessibility of the Date and Time Pickers.
+
+## Screen reader compatibility
+
+Date and Time Pickers provide support to ARIA roles and a robust focus management across the interactive elements to convey the necessary information to users, being optimized for use with assistive technologies.
+
+## Keyboard support
+
+The Date and Time Pickers consist on different associations of Fields, Calendar, Time and Digital Clock components, each of them designed to respond intuitively to keyboard interactions, providing a better experience for users who rely on keyboard navigation.
```

Comment:
"consist of"? Maybe?

Reply:
The sentence is very long. Can we split it up into at least 2 sentences?

Reply:
Done in  [8e2b7e3](https://github.com/mui/mui-x/pull/13274/commits/8e2b7e3de0c9884702a21f75b5b00c1dc4c51a5f)